pypeit/io.py

```diff
@@ -830,4 +830,54 @@ def fits_open(filename, **kwargs):
         return fits.open(filename, ignore_missing_end=True, **kwargs)
 
 
+def create_symlink(filename, symlink_dir, relative_symlink=False, overwrite=False, quiet=False):
```

Comment:
for future reference, i think this could be implemented a bit more cleanly with `pathlib`.

Reply:
Good tip.  I've only recently used `pathlib` for another project.  I'd be interested to hear more about any particular pros/cons wrt `os`.

Reply:
the main pro is that it treats paths as objects rather than just strings. that makes getting things like the parent, stem, name, and suffix a lot easier. and there's a / operator for joining paths or joining a string to a path to make a new path. there are several comparison write-ups. this is one of the better ones: https://medium.com/aiadventures/why-you-should-always-use-pathlib-5212accde268

Reply:
Thanks!  I'll have a look.